depends/packages/twisted.mk

```diff
@@ -0,0 +1,10 @@
+package=twisted
+$(package)_version=15.0.0
+$(package)_download_path=https://pypi.python.org/packages/source/T/Twisted
+$(package)_file_name=Twisted-$($(package)_version).tar.bz2
+$(package)_sha256_hash=ac609262253057cf2aeb9dc049ba7877d646f31b4caef06a50189a023df46b51
+$(package)_dependencies=native_python2 native_zope.interface
+
+define $(package)_stage_cmds
+  $($(package)_prefix)/native/bin/python setup.py install --prefix=$($(package)_staging_prefix_dir)
```

Comment:
It'll matter if bytecode is created. Ugly as it sounds, you'll likely end up needing to build 2 pythons, 1 native and 1 host. That's how I've dealt with this in the past.

As for the toolchain stuff, here's how I changed sip as an example (not 100% correct I'm sure, I was just playing)
```
package=native_sip
$(package)_version=4.16.7
$(package)_download_path=http://sourceforge.net/projects/pyqt/files/sip/sip-$($(package)_version)
$(package)_file_name=sip-$($(package)_version).tar.gz
$(package)_sha256_hash=4caa8d52e4403bae5c4c64f44de03a2cfd0bb10b6d96c5fb771133516df9abbd
$(package)_dependencies=native_python2

define $(package)_set_vars
  $(package)_config_opts= CC="$($(package)_cc)" CXX="$($(package)_cxx)" CFLAGS="$($(package)_cflags)"
  $(package)_config_opts+=CXXFLAGS="$($(package)_cxxflags)" LFLAGS="$($(package)_ldflags)"
  $(package)_config_opts+=LINK_SHLIB="$($(package)_cc)" LINK="$($(package)_cc)"
endef

define $(package)_config_cmds
  $($(package)_prefixbin)/python configure.py $($(package)_config_opts)
endef

define $(package)_build_cmds
  $(MAKE)
endef

define $(package)_stage_cmds
  $(MAKE) DESTDIR=$($(package)_staging_dir) install
endef
```
and psutil:
```
package=native_psutil
$(package)_version=2.2.1
$(package)_download_path=https://pypi.python.org/packages/source/p/psutil
$(package)_file_name=psutil-$($(package)_version).tar.gz
$(package)_sha256_hash=a0e9b96f1946975064724e242ac159f3260db24ffa591c3da0a355361a3a337f
$(package)_dependencies=native_python2
$(package)_config_env=CC="$($(package)_cc)" LD="$($(package)_cc)" CFLAGS="$($(package)_cflags)"

define $(package)_build_cmds
  $($(package)_prefixbin)/python setup.py build
endef

define $(package)_stage_cmds
  $($(package)_prefixbin)/python setup.py install --root=$($(package)_staging_dir) --prefix=$($(package)_prefix)
endef
```